pages/osx/cot.md

```diff
@@ -0,0 +1,24 @@
+# cot
+
+> The Plain-Text Editor for macOS.
+> More information: <https://coteditor.com/>.
+
+- Start CotEditor:
+
+`cot`
+
+- Open specific files in the background:
+
+`cot {{path/to/file1 path/to/file2 ...}}`
+
+- Open a new blank document in the background:
+
+`cot --new`
+
+- Open a specific file (blocking the terminal until the editor is closed):
```

Comment:
```suggestion
- Open a specific file in the foreground:
```

Reply:
I would be fine with that, but I just have the concern that it's not really clear, what `in the foreground` means here.

Reply:
`in the background` expression also can be confusing. :) Maybe it's better to write what `in the background` actually means such as `without blocking a terminal until the editor is closed`.

Reply:
That's why I went for this solution, which seemed most natural to me. However, that's just one opinion. I'd say in cases like this we should let the author decide (@superatomic).

Reply:
I was interpreting foreground/background to be whether the window that was opened was sent to the foreground, not whether the terminal blocks.

As it currently stands, either interpretation is correct, although it's probably better to use more descriptive language to avoid confusion.

For this line, I personally like "Open and wait until finished editing a specific file", which was the original text before it was changed. I'd suppose "Open a specific file and wait until it is closed" might be a good way to phrase this as well.

Reply:
I like the last suggestion. But `wait` is a bit ambiguous. How about `Open a specific file and block the terminal until it is closed`?

Reply:
That sounds good. I chose the word "wait" because it was the name of the option, but I like yours more.